validator/tests/unit/test_signed_object.py

```diff
@@ -78,8 +76,7 @@ def test_signed_node(self):
         # Verify that signed_node and sign_object does not invalidate the
         # signed object and can be returned to original
         # create initial signed object
-        signkey = SigObj.generate_signing_key()
-        temp = SignedObject({signkey: "test"}, signkey)
+        temp = SignedObject({"TestSignatureName": "test"}, "TestSignatureName")
```

Comment:
You simplified some of these tests. Looks good to me, but are you sure you didn't break anything by simplifying this tests and the ones below? If you are confident, I'll merge the PR after the minor change above.